### PR TITLE
Updating documentation / PR template to target **version-3** of PnPjs

### LINKED ISSUE #2613: [Documentation] Update contribution guide / sample / links to version-3
### Category
- [ ] Enhancement
- [ ] Bug
- [ ] Question
- [x] Documentation gap/issue

### Version

Please specify what version of the library you are using: [N/A]

Please specify what version(s) of SharePoint you are targeting: [N/A]

### Expected / Desired Behavior / Question

Contributing documentation / template should target **version-3** of PnPjs.

### Observed Behavior

Actual contributing documentation / target is targeting **version-2** of PnPjs.

### Steps to Reproduce

N/A

### PR DESCRIPTION
**Pull request by michaelmaillot**
#### Category
- [ ] Bug fix?
- [ ] New feature?
- [ ] New sample?
- [x] Documentation update?

#### Related Issues

fixes #2613 

#### What's in this Pull Request?

Update contributing documentation (links / v3 mentions) and PR template to target latest version of PnPjs

#### Guidance

*You can delete this section when you are submitting the pull request.*

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ fixes #X, mentioned in #Y
 *You can delete this section when you are submitting the pull request.* 
 
 * Please update this PR information accordingly. We'll use this as part of our release notes in monthly communications.
-* Please target your PR to "version-2" branch for v2 changes and "version-1" branch for v1 changes
+* Please target your PR to "version-3" branch for v3 changes, "version-2" branch for v2 changes and "version-1" branch for v1 changes
```
Comment (bcameron1231) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Let's remove references to V2 and V1, as we are no longer supporting these branches or doing releases.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+* Target your pull requests to the **version-3** branch
+* Add/Update any docs articles related to your changes
+* Include a test for any new functionality and ensure all existing tests are passing by running `npm test`
+  * If you are fixing a bug, include a test that would have caught the bug you are fixing
+* Ensure tslint checks pass by typing `npm run lint`
+* Keep your PRs as simple as possible and describe the changes to help the reviewer understand your work
+* If you have an idea for a larger change to the library please [open an issue](https://github.com/pnp/pnpjs/issues) and let's discuss before you invest many hours - these are very welcome but want to ensure it is something we can merge before you spend the time :)
+
 ## Contributing Guidance - v2
```
Comment (bcameron1231) [inline CONTRIBUTING.md:11]: Let's remove references to V2, as we are no longer supporting v2 or publishing any new releases.

**Review by bcameron1231 — CHANGES_REQUESTED** [review]
Thanks Michael! Just a couple changes requested.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (michaelmaillot) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: No pb, will remove!

File: CONTRIBUTING.md (hunk 2 above)
Comment (michaelmaillot) [reply CONTRIBUTING.md:11]: Shouldn't I also remove v1 references?

Comment (bcameron1231) [reply CONTRIBUTING.md:11]: Yes please. :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (bcameron1231) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: Thank you! :)

**Comment by michaelmaillot** [comment]
Thx for your review, I've commited requested changes 🙂

**Comment by bcameron1231** [comment]
Thanks!

**Pull request merged**